test/functional/core/job_spec.lua

```diff
@@ -97,6 +97,28 @@ describe('jobs', function()
     command("call jobstart(['cat', '-'], { 'term': v:false })")
   end)
 
+  it('jobstart(term=true) accepts width/height (#33904)', function()
+    local buf = api.nvim_create_buf(false, true)
+    exec_lua(function()
+      vim.api.nvim_buf_call(buf, function()
+        vim.fn.jobstart({
+          vim.v.progpath,
+          '--clean',
+          '--headless',
+          '+lua print(vim.uv.new_tty(0, true):get_winsize())',
```

Comment:
this is failing on Windows CI. maybe try 1 instead of 0, idk. `new_tty` is not used in any of our tests. could some other mechanism be used?

```
FAILED test/functional/core/job_spec.lua @ 100: jobs jobstart(term=true) accepts width/height (#33904) test/functional\core\job_spec.lua:117: retry() attempts: 324 test/functional\core\job_spec.lua:118: Expected objects to be the same. Passed in: (table: 0x027ef06a8588) { *[1] = 'nil EBADF: ' } Expected: (table: 0x027ef06ad650) { *[1] = '11 12' } stack traceback: test\testutil.lua:89: in function 'retry' test/functional\core\job_spec.lua:117: in function <test/functional\core\job_spec.lua:100>
```